### ISSUE TITLE
``unsubscriptable-object`` in numpy array method

### ISSUE THREAD

**Issue by Teagum** [PyCQA/pylint#7981]
### Bug description

Pylint reports a false-positive error message, when indexing the return value of numpy's `strides` method. The error does *not* occure if your remove the call to `np.pad`. However, as per the [numpy documentation](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.strides.html), the return value of `arr.strides` is always a tuple. A zero-sized array, for example, has strides `(0, )`.
```
# pylint disable=missing-docstring

import numpy as np

arr = np.arange(10)
arr = np.pad(arr, (2, 4))
print(arr.strides[0])
```

### Command used

```shell
pylint c.py
```

### Pylint output

```shell
************* Module c
c.py:6:6: E1136: Value 'arr.strides' is unsubscriptable (unsubscriptable-object)
```

### Expected behavior

Don't report error

### Pylint version

```shell
ylint 2.15.9
astroid 2.12.13
Python 3.10.8 (main, Nov 25 2022, 23:50:20) [Clang 13.0.0 (clang-1300.0.29.30)]
```

### Additional dependencies

numpy==1.24.0
